openquake/risklib/__init__.py

```diff
@@ -33,3 +33,5 @@
     sys.exit('Found an obsolete version of commonlib; '
              'please remove %s and/or fix your PYTHONPATH'
              % os.path.dirname(path))
+
+__codename__ = 'Keiiti Aki'
```

Comment:
We should keep the `__codename__` empty and populate it only during the release process.